src/Output.mli

```diff
@@ -0,0 +1,5 @@
+(** [languageServerOutputChannel] is the output channel that should be used by all ocamllsp LanguageClients *)
+val languageServerOutputChannel : Vscode.Window.OutputChannel.t
+
+(** [extensionOutputChannel] is the output channel that should be used for logs by the extension *)
+val extensionOutputChannel : Vscode.Window.OutputChannel.t
```

Comment:
Can we make both values in this module lazy? It's better to avoid module side effects when possible.